app/scripts/lib/accounts/Poller.ts

```diff
@@ -0,0 +1,24 @@
+export class Poller {
+  #interval: number;
+
+  #callback: () => void;
+
+  #handle: NodeJS.Timeout | undefined = undefined;
+
+  constructor(callback: () => void, interval: number) {
+    this.#interval = interval;
+    this.#callback = callback;
+  }
+
+  start() {
+    this.stop();
+    this.#handle = setInterval(this.#callback, this.#interval);
```

Comment:
This logic differs from the (multichain) `RatesController`, if the you `start` again, we check for the handler, if it exists, then we do nothing. Should we do the same here?